app/services/diaper_partner_client.rb

```diff
@@ -70,9 +70,10 @@ def self.https(uri)
 
   def self.partner_json(attributes, invitation_message)
     { partner:
-          { diaper_bank_id: attributes["organization_id"],
-            diaper_partner_id: attributes["id"],
-            invitation_text: invitation_message,
-            email: attributes["email"] } }.to_json
+      { diaper_bank_id: attributes["organization_id"],
+        diaper_partner_id: attributes["id"],
+        invitation_text: invitation_message,
+        email: attributes["email"],
+        organization_email: attributes["organization_email"] } }.to_json
```

Comment:
I'm a little concerned that we might get some breakage with this -- we don't require organizations to have an email so it is possible that we could be sending and then having the reply_to be an empty string. I imagine we could grab an email from one of the users of the organization if it was empty?

Reply:
@seanmarcia good point. I figured that could happen and it wasn't certain that the `organization_email` would be defined. I added a fallback on the partner side that sets it back to the original value before this PR - https://github.com/rubyforgood/partner/pull/309/files#diff-54b58671138880a646ba8e9551691067R20

Let me know if that covers this case?

Reply:
Cool! That looks like a great solution to me! :)

Reply:
In a pinch, could always use `fetch` -- `attributes.fetch('organization_email', '')`?

Reply:
Ahh its better off to be nil in this case, since in the other code it uses `organization_email_received || fallback` and when `organization_email_received` is an empty string it won't fallback. I can make it go fall back to nil though :)